.github/workflows/nm-remote-push.yml

```diff
@@ -12,80 +12,18 @@ concurrency:
 
 jobs:
 
-    BUILD-TEST-3-8:
-        uses: ./.github/workflows/nm-build-test.yml
-        with:
-            python: 3.8.17
-            gitref: ${{ github.ref }}
-            push_binaries_to_pypi: 'no'
-
-            test_label_solo: gcp-k8s-l4-solo
-            test_label_multi: ignore
-            test_timeout: 480
-            test_skip_env_vars: neuralmagic/tests/test_skip_env_vars/smoke.txt
-
-            benchmark_label: gcp-k8s-l4-solo
-            benchmark_config_list_file: ./.github/data/nm_benchmark_remote_push_configs_list.txt
-            benchmark_timeout: 480
-
-            lm_eval_label: gcp-k8s-l4-solo
-            lm_eval_configuration: ./neuralmagic/lm-eval/smoke-small-models.yaml
-            lm_eval_timeout: 60
-        secrets: inherit
-
-    BUILD-TEST-3-9:
-        uses: ./.github/workflows/nm-build-test.yml
-        with:
-            python: 3.9.17
-            gitref: ${{ github.ref }}
-            push_binaries_to_pypi: 'no'
-
-            test_label_solo: gcp-k8s-l4-solo
-            test_label_multi: ignore
-            test_timeout: 480
-            test_skip_env_vars: neuralmagic/tests/test_skip_env_vars/smoke.txt
-
-            benchmark_label: gcp-k8s-l4-solo
-            benchmark_config_list_file: ./.github/data/nm_benchmark_remote_push_configs_list.txt
-            benchmark_timeout: 480
-
-            lm_eval_label: gcp-k8s-l4-solo
-            lm_eval_configuration: ./neuralmagic/lm-eval/smoke-small-models.yaml
-            lm_eval_timeout: 60
-        secrets: inherit
-
-    BUILD-TEST-3-10:
+    REMOTE:
         uses: ./.github/workflows/nm-build-test.yml
         with:
             python: 3.10.12
```

Comment:
Should we remove the python version here too?

Reply:
nope, same reason as in the nightly.